src/components/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,33 @@
+import './ComingSoon.css'
+
+
+function ComingSoon ()
+{
+    return ( 
+        <div className='center'>  
+<h1 className='google-text-1'>
+    <span className = 'text-google-blue'>C</span>
+    <span class = "text-google-red">o</span>
+    <span class = "text-google-yellow">m</span>
+    <span class = "text-google-blue">i</span>
+    <span class = "text-google-green">n</span>
+    <span class = "text-google-red">g </span>
+    <span class = "text-google-green"> S</span>
+    <span class = "text-google-yellow">o</span>
+    <span class = "text-google-red">o</span>
+    <span class = "text-google-blue">n</span>
+    <span class = "text-google-yellow">.</span>
+    <span class = "text-google-green">.</span>
+    <span class = "text-google-red">.</span>
+</h1>   
+<br/>  
+<h2 className='google-text-2 text-google-blue'> 
+This page is currently under development and we hope it will be completed soon!
+</h2>   
+
+<p className='google-text-3 text-align-center text-google-green'> If you're interested in seeing our progress you can check our <span> <a className='text-google-yellow' href='https://github.com/gdsc-conestoga/website'> GitHub.</a> </span></p>
```

Comment:
Make sure tag spacing is consistent. I will add a linter a bit later, but for now we'll need to do this manually